### ISSUE TITLE
make statusstylepretty symbols/filters configurable 

### ISSUE THREAD

**Issue by bovine3dom** [tridactyl/tridactyl#4626]
TheSirC on matrix asks that the tab status symbols like % and # become configurable because those symbols aren't easy to type on some keyboard layouts

**Comment by TheSirC** [comment]
If I read the code correctly, this is the part (the string constant `pre` and `preplain`) that one should be able to parameterize ?

https://github.com/tridactyl/tridactyl/blob/e3b44a69a9729ff8903a72b004c3bf6a44d44432/src/completions/Tab.ts#L29-L48

Correct ?

**Comment by bovine3dom** [comment]
Yeah, but I can't promise that's the only place in the code that #/% are defined. I suspect there are other places

**Comment by TheSirC** [comment]
It seems it is also in :
https://github.com/tridactyl/tridactyl/blob/e3b44a69a9729ff8903a72b004c3bf6a44d44432/src/completions/TabGroup.ts#L25-L42
